apps/example/src/app/RudderEvents.tsx

```diff
@@ -108,9 +108,12 @@ const RudderEvents = () => {
     // rudderClient.alias('New User ID', null);
     // rudderClient.alias('New User ID', undefined);
     // rudderClient.alias("New User ID", getLocalOptions());
-
-    // Deprecated
-    // rudderClient.alias('Old User ID', 'New User ID');
+    // rudderClient.alias('New User ID', null, null);
+    // rudderClient.alias('New User ID', null, undefined);
+    // rudderClient.alias('New User ID', null, getLocalOptions());
+    // rudderClient.alias('New User ID', undefined, getLocalOptions());
+    // rudderClient.alias('New User ID', 'Old User ID', null);
+    // rudderClient.alias('New User ID', 'Explicit Previous UserId', getLocalOptions());
```

Comment:
Let's simplify this to only the actual legal usage of the API.

For example,
`rudderClient.alias('New User ID');` instead of `rudderClient.alias('New User ID', null, null);`.
`rudderClient.alias('New User ID', getLocalOptions());` instead of `rudderClient.alias('New User ID', undefined, getLocalOptions());`